### ISSUE TITLE
EntityModel cannot be generated from a simple Threejs geometry

### ISSUE THREAD

**Issue by meszaros-lajos-gyorgy** [arx-tools/arx-level-generator#5]
```ts
import { Entity, EntityConstructorPropsWithoutSrc, EntityModel, Texture, Vector3 } from 'arx-level-generator'
import { BoxGeometry, Mesh, MeshBasicMaterial } from 'three'

const geometry = new BoxGeometry(1, 1, 1)
const material = new MeshBasicMaterial({ map: Texture.l2TrollStoneGround04 })
const mesh = new Mesh(geometry, material)

export class Pillar extends Entity {
  constructor({ ...props }: EntityConstructorPropsWithoutSrc = {}) {
    super({
      src: 'fix_inter/pillar',
      model: EntityModel.fromThreeJsObj(mesh, {
        filename: 'pillar.ftl',
      }),
      ...props,
    })
  }
}
```

This results in the following error message in game and the model not showing:

```
[E] FTL:179               "game/graph/obj3d/interactive/fix_inter/pillar/pillar.ftl": Invalid face material
```

Extra infos:

* otherDependencies is not needed in this case as an in-game texture is being used
* works correctly if an obj file is loaded and passed to the `fromThreeJsObj` method
* don't forget when debugging that generated ftl files are cached in the `cache` directory

**Comment by meszaros-lajos-gyorgy** [comment]
The issue comes from `getNonIndexedVertices` returning something called `materialIndex` which is in fact a group index of the groups inside the geometry. The number of materials don't match the material indices.

**Comment by meszaros-lajos-gyorgy** [comment]
Fixed in `13.0.0-alpha.2`
